src/main/java/core/basesyntax/service/serviceimpl/CheckFilePathImpl.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.service.interfaces.CheckFilePath;
+
+public class CheckFilePathImpl implements CheckFilePath {
+    @Override
+    public boolean checkFilePath(String filePath) {
+        return (filePath == null || filePath.isEmpty());
+    }
```

Comment:
do we really need a whole class for that?

Reply:
Well I sued this check in two other classes and planned to use in tests but I guess there is no need of it now)